### ISSUE TITLE
IO Error: Bad chunk header - reported inconsistently

### ISSUE THREAD

**Issue by uolpas** [validator/validator#1242]
URL being validated or code to reproduce error:
https://www.leeds.ac.uk/
```
    IO Error: Bad chunk header: ì}érãF²îo
```

The web nu checker reports this error at https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.leeds.ac.uk%2F but you don't get this error testing the http url (https://validator.w3.org/nu/?doc=http%3A%2F%2Fwww.leeds.ac.uk%2F), even though this just redirects to https.

Testing with the command line version (vnu https://www.leeds.ac.uk) also doesn't show this error.

Is this an issue with with https://validator.w3.org/nu/ ?

**Comment by sideshowbarker** [comment]
> The web nu checker reports this error at [validator.w3.org/nu/?doc=https%3A%2F%2Fwww.leeds.ac.uk%2F](https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.leeds.ac.uk%2F)
> …
> Is this an issue with with [validator.w3.org/nu](https://validator.w3.org/nu/) ?

I could reproduce this previously but I can’t now.

Did you change something on the https://www.leeds.ac.uk/ server in the meantime?

If not, then I guess it’s possible that maybe it was fixed by some other recent changes I made to the checker.

**Comment by paasher** [comment]
I'm still seeing the same behaviour, bad chunk header error for https but not for http. Had cached results at one stage but hard refresh showed it hadn't changed.

**Comment by sideshowbarker** [comment]
The checker HTTP client now always makes HTTP 1.0 requests — rather than HTTP 1.1 ones — which means it will no longer be getting any chunked responses. Which in turn means that this error should no longer occur.

So I’m going ahead and closing this.
